controller/src/main/java/com/softserve/borda/controllers/UserController.java

```diff
@@ -131,12 +120,13 @@ public UserSimpleDTO updateUser(@RequestBody final UserUpdateDTO userDTO, @PathV
         }
     }
 
-    @GetMapping("{userId}/boardsByRole/{boardRoleId}")
-    public ResponseEntity<List<BoardFullDTO>> getBoardsByUserIdAndBoardRoleId(@PathVariable Long userId,
+    @GetMapping("/boardsByRole/{boardRoleId}")
```

Comment:
Bad resource naming.
Please read https://restfulapi.net/resource-naming/ doc and change naming.

Reply:
Fixed in FixResourcesPaths branch/ticket, already merged in main